kingpin/actors/rightscale/server_array.py

```diff
@@ -946,7 +946,11 @@ class Launch(ServerArrayBaseActor):
             "Number of server to launch. Default: up to array's min count"),
         'enable': (bool, False, 'Enable autoscaling?'),
         'exact': (bool, True, (
-            'Whether to search for multiple ServerArrays and act on them.')),
+            'Whether to search for multiple ServerArrays and act on them.'),),
+        'replace_stranded_max': (
+            (int, str), 0,
+            "Maximum number of times to launch replacements for stranded servers."
```

Comment:
This isn't accurate. If you're launching 10 machines and 2 get stranded, and your `replace_stranded_max` is set to 2 then no replacements will be tried